### ISSUE TITLE
Can't install a mod from a file

### ISSUE THREAD

**Issue by gurleygirl** [Nexus-Mods/Nexus-Mod-Manager#1003]
When I download a mod from a file it does not show up in the list of available mods which can be installed. 

**Environment**
- NMM version: 0.71.3
- Mod(s) used: Skyrim SSE Water for ENB : http://enbseries.enbdev.com/forum/viewtopic.php?f=6&t=6480

NMM didn't crash so I don't think there is an applicable trace log.

**To Reproduce**
Simply try to install a mod from a file. After downloading it you will not be able to see the mod as available to be installed.

**Comment by squid-box** [comment]
I'm not sure I follow, I've tried to download this file (Skyrim SSE Water for ENB.zip), and adding it to NMM 0.71.3 works fine for me - I can install it without issue.

Are you getting an error message? If not, the only way for us to figure out what's going on is a trace log. [You can force the creation of one](https://github.com/Nexus-Mods/Nexus-Mod-Manager/wiki/Bug-reporting#trace-log).

**Comment by gurleygirl** [comment]
No, I'm not getting an error. NMM downloads the file but I don't see the file listed as available for installation. I have sorted by Name and sorted by Install Date and I simply can't see the downloaded mod so that I can install it. I restarted NMM to no avail. 
But since it is working for you, then it must be something odd on my end. 
I will reboot my PC aka Windows 10 to see if that makes a difference but honestly I don't know why it would. FYI, I did PC Support for a large company for 12 years so I'm pretty computer savvy but that doesn't mean I'm not overlooking something obvious. hahaa

OK, I will restart NMM with the /trace switch and will do the Install Mod from File again but I will be surprised if the log provides the info you need to troubleshoot the issue. Again, NMM, downloads the file just fine but I simply can't see the downloaded grayed out mod listed as available for installation.

**Comment by gurleygirl** [comment]
OK, I found the problem: user error which is usually the case. :)
The name of the file is Skyrim SSE Water for ENB so I was looking for the mod listing based on the file name when in fact the name of the mod is Water for ENB which is how it was listed in NMM. 
My apologies for the wild goose chase. 
For future reference, I discovered the mod name/listing by sorting by Download Date in descending order to see the most recent download. <face palm>
